### PR TITLE
10/18

### PR DESCRIPTION
**Pull request by timsehn**

### CODE REVIEW

File: emails/2024-10-18.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+I've been using the Dolt Workbench instead of the command line shell and I really enjoy the experience. It's like Datagrip and Git had a baby and matured through 20 years of GUI development.
+
+### Is MySQL Dying?
+
+I'm not sure if many of you saw but the commercial arm of MariaDB was [acquired at a massive discount](https://techcrunch.com/2024/09/10/mariadb-goes-private-with-new-ceo-as-k1-closes-acquisition/) to their initial offering price two years ago. This prompted me to write an [Is MySQL Dying? article](https://www.dolthub.com/blog/2024-10-14-is-mysql-dying/). 
+
+I'm of the opinion that AWS is draining all the profits from the MySQL ecosystem and this makes Oracle mad. And frankly, I don't blame Oracle. But, it's hurting the ecosystem, potentially mortally. Good thing [Doltgres is almost Beta](https://www.dolthub.com/blog/2024-08-06-doltgres-beta/). Don't worry. We'll continue to stand for MySQL as long as we have people using Dolt.
```
Comment (zachmu) [inline emails/2024-10-18.md:23]: stan?

**Review by zachmu — APPROVED** [review]
LG

File: emails/2024-10-18.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+I've been using the Dolt Workbench instead of the command line shell and I really enjoy the experience. It's like Datagrip and Git had a baby and matured through 20 years of GUI development. We also just [announced Workbench support for Doltgres]() so use it with either.
+
+### Is MySQL Dying?
+
+I'm not sure if many of you saw but the commercial arm of MariaDB was [acquired at a massive discount](https://techcrunch.com/2024/09/10/mariadb-goes-private-with-new-ceo-as-k1-closes-acquisition/) to their initial offering price two years ago. This prompted me to write an [Is MySQL Dying? article](https://www.dolthub.com/blog/2024-10-14-is-mysql-dying/). 
+
+I'm of the opinion that AWS is draining all the profits from the MySQL ecosystem and this makes Oracle mad. And frankly, I don't blame Oracle. But, it's hurting the ecosystem, potentially mortally. Good thing [Doltgres is almost Beta](https://www.dolthub.com/blog/2024-08-06-doltgres-beta/). Don't worry. We'll continue to Stan for MySQL as long as we have people using Dolt.
```
Comment (macneale4) [inline emails/2024-10-18.md:23]:
```suggestion
I'm of the opinion that AWS is draining all the profits from the MySQL ecosystem and this makes Oracle mad. And frankly, I don't blame Oracle. But, it's hurting the ecosystem, potentially mortally. Good thing [Doltgres is almost Beta](https://www.dolthub.com/blog/2024-08-06-doltgres-beta/). Don't worry. We'll continue to stand for MySQL as long as we have people using Dolt.
```

Comment (timsehn) [reply emails/2024-10-18.md:23]: Stan is slang Neil. 

https://www.merriam-webster.com/dictionary/stan

There's a whole Dido + Eminem song.

**Pull request merged**